### ISSUE TITLE
Cannot validate resource with no exported attributes

### ISSUE THREAD

**Issue by manicminer** [bflad/tfproviderdocs#43]
Docs for resources that don't have, or wish to, document any exported attributes, since they are identical to the arguments, cannot be validated by `tfproviderdocs check`.

If no `## Attributes Reference` header exists, it reports:
```
* docs/resources/application_app_role.md: error checking file contents: missing attributes section: ## Attributes Reference
```

If the header is included but has alternative paragraph text:
```
* docs/resources/application_app_role.md: error checking file contents: attributes section byline (No additional attributes are exported.) should be: In addition to all arguments above, the following attributes are exported:
```

Could this check be made togglable so that the docs can still be otherwise checked?

**Comment by bflad** [comment]
Just for the brief history here: this type of checking was originally created as every Terraform Plugin SDK resource implicitly had an `id` attribute, whether or not the schema declares it. `terraform-plugin-go` resources no longer have this particular behavior.

I think the preference here would be to always check for the header, since it would be good to check for accidental omissions. However, we could allow an additional byline similar to the one you mention: `No additional attributes are exported.`

Would this be sufficient or would you prefer the bylines be configurable?

**Comment by manicminer** [comment]
Hey @bflad, that makes total sense and it's logical that resources would _typically_ set at least one attribute. I guess greater prescription makes more sense and personally I'd still want to include the heading so that it doesn't just look like it was forgotten. Allowing an alternative byline like that would be great for this sort of case 👍

**Comment by bflad** [comment]
Awesome -- should be relatively straightforward to accept the alternate line by default. 😄

**Comment by bflad** [comment]
Released in v0.9.1 🚀
